.circleci/config.yml

```diff
@@ -93,69 +93,22 @@ jobs:
       
   deploy-doc:
     docker:
-      - image: cimg/node:14.0.0  # npm required for gh-pages
-
+      - image: cimg/python:3.8.12
     steps:
       - checkout
- 
       - add_ssh_keys:
-          # To generate a key run the following in git bash (on Windows):
-          # ssh-keygen -t rsa -m pem -C "ci-build@fairlearn.org"
```

Comment:
this is not very secure though, better create a ed25519 key. It's supported, even though implied otherwise.